plugins/katello/developers.md

```diff
@@ -230,35 +226,19 @@ Create a 2.1 release configuration in tool_belt at `configs/katello/2.1.yaml`. Y
 
 You will find the repositories checked out to the versions specified in the release config in `repos/katello/2.1.0/`.
 
-### Step 5: Get Your Branch On
-
-#### Step 5.1: Create the branches for gems and docs
-
-On the date you previously warned the community branching would occur, ensure any project dependencies are updated, including gem version constraints, and branch the repositories:
+The `tool_belt` repository contains templated procedures that can be generated and then posted to Discourse for tracking of all release activities. To generate:
 
 ```
-./tools branch configs/katello/2.1.yaml
+./tools.rb procedure branch katello 2.1
+./tools.rb procedure release katello 2.1
 ```
 
-You will also need to branch the documentation at [theforeman.org](https://theforeman.org/plugins/katello).
+Copy the output to a Discourse post in the [release section](https://community.theforeman.org/c/development/releases) titled 'Katello 2.1 Release Process'
 
-```
-./tools branch-docs config/katello/2.1.yaml
-```
-
-Make sure to follow the instructions and update the docs to use the current release version. Submit a pull request to theforeman.org repository with the changes.
-
-Congratulations, you have successfully branched all of the repositories and prepared them for release. We recommend double checking each repository ensuring that the branch exists in the upstream repository, that the top of the commit stack in that branch includes the tag you created prior to branching (so that you have an initial tag point that represents the branch point) and that the tag itself exists. The last part of this step is to send to a follow up notification letting the community know that everything has been branched.
-
-#### Step 5.2: Teach Jenkins about the new branch
-
-Jenkins has a [branch map](https://github.com/theforeman/foreman-infra/blob/master/puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/test/testKatello.groovy#L2) for running Katello tests against the correct Foreman version, this map needs to be updated to contain the new version you've just branched.
+From here on out the release owner and release packager should work to close out each of the respective item lists.
 
-### Step 6. Release puppet modules
 
-Review the modules for releasing - we should be keeping them up to date when making incompatible changes, but do any need bumps of the major or minor version? If so, do so now.
-
-Go through the [list of managed modules](https://github.com/theforeman/foreman-installer-modulesync/blob/master/managed_modules.yml).
+## How to Release puppet modules
```

Comment:
Now that the installer has been merged, this is part of the Foreman release process. Since it's not documented elsewhere, I'm fine keeping it in (with a note).